core/trino-main/src/main/java/io/trino/connector/ConnectorManager.java

```diff
@@ -466,16 +466,13 @@ public MaterializedConnector(CatalogName catalogName, Connector connector)
             this.catalogName = requireNonNull(catalogName, "catalogName is null");
             this.connector = requireNonNull(connector, "connector is null");
 
-            Set<SystemTable> systemTables = connector.getSystemTables();
-            requireNonNull(systemTables, format("Connector '%s' returned a null system tables set", catalogName));
+            Set<SystemTable> systemTables = requireNonNull(connector.getSystemTables(), format("Connector '%s' returned a null system tables set", catalogName));
```

Comment:
Doing this inline makes it harder to read the code. The existing structure is intentional.